### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by caozhijianTest**
what you need to update is already finished.
Please check as quickly as much
[新建 Microsoft Word 文档.docx](https://github.com/CaoZhijian1123/DrawMap/files/3825315/Microsoft.Word.docx)

### CODE REVIEW

**Comment by CaoZhijian1123** [comment]
你确定？？

**Comment by caozhijianTest** [comment]
确定

**Comment by CaoZhijian1123** [comment]
真的

**Comment by caozhijianTest** [comment]
66666

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
 # DrawMap
 This project is used to generate map of traffic attraction distribution.
+
+## git reset --[<mode>]  [index]
```
Comment (CaoZhijian1123) [inline README.md:4]: 123

**Pull request merged**